src/test/java/com/booking/replication/schema/column/types/ConverterTest.java

```diff
@@ -0,0 +1,276 @@
+    // INTEGER CELLS
+
+    @Test
+    public void tinyintSignedCell() throws TableMapException {
+        int x = -125; // inside single byte
+        Cell c = TinyCell.valueOf(x);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("tinyint");
+        s.setDataType("tinyint"); // TODO : Remove
+
+        assertEquals(Integer.toString(x), Converter.cellValueToString(c, s));
+    }
+
+    // TODO: Failing!!!
+//    @Test
+//    public void tinyintUnsignedCell() throws TableMapException {
+//        int x = 225; // inside single byte
+//        Cell c = TinyCell.valueOf(x);
+//        ColumnSchema s = new ColumnSchema();
+//        s.setColumnType("unsigned tinyint");
+//        s.setDataType("tinyint"); // TODO : Remove
+//
+//        assertEquals(Integer.toString(x), Converter.cellValueToString(c, s));
+//    }
+
+    @Test
+    public void smallintCell() throws TableMapException {
+        int x = -30000;
+        Cell c = ShortCell.valueOf(x);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("smallint");
+        s.setDataType("smallint"); // TODO: Remove
+
+        assertEquals(Integer.toString(x), Converter.cellValueToString(c, s));
+    }
+
+    @Test
+    public void smallintUnsignedCell() {
+        // TODO
+    }
+
+    @Test
+    public void mediumintCell() throws TableMapException {
+        int x = -2000000;
+        Cell c = Int24Cell.valueOf(x);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("mediumint");
+        s.setDataType("mediumint"); // TODO: Remove
+
+        assertEquals(Integer.toString(x), Converter.cellValueToString(c, s));
+    }
+
+    @Test
+    public void mediumintUnsignedCell() {
+        // TODO
+    }
+
+    @Test
+    public void intCell() throws TableMapException {
+        int x = -2000000000;
+        Cell c = LongCell.valueOf(x);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("int");
+        s.setDataType("int");
+
+        assertEquals(Integer.toString(x), Converter.cellValueToString(c, s));
+    }
+
+    @Test
+    public void intUnsignedCell() {
+        // TODO
+    }
+
+    @Test
+    public void bigintCell() throws TableMapException {
+        long x = -9000000000000000000L;
+        Cell c = LongLongCell.valueOf(x);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("bigint");
+        s.setDataType("bigint");
+
+        assertEquals(Long.toString(x), Converter.cellValueToString(c, s));
+    }
+
+    @Test
+    public void bigintUnsignedCell() {
+        // TODO
+    }
+
+    // TODO : Add tests for overflows
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // REAL NUMBERS
+
+    @Test
+    public void doubleCell() throws TableMapException {
+        Cell c = new DoubleCell(1.5);
+        ColumnSchema s = new ColumnSchema();
+
```

Comment:
Here the setting of the column type and data type for the  ColumnSchema object is missing.